package.json

```diff
@@ -146,8 +146,7 @@
             },
             "additionalProperties": false,
             "required": [
-              "serverUrl",
-              "token"
+              "serverUrl"
```

Comment:
What does this do?

Reply:
Since we migrating tokens from settings.json file to secret storage we don't ask for token as required field anymore.